docs/docs/quickstart.md

```diff
@@ -95,17 +97,19 @@ UUID   Name   Ports   Status
 
 Congratulations - you've written your first Kurtosis code!
 
-### Review
+### Review: Hello, World
 :::info
 We'll use these "Review" sections to explain what happened in the section. If you just want the action, feel free to skip them.
 :::
 
-In this section, we created a `.star` file that prints `Hello, world`. The `.star` extension corresponds to [the Starlark language developed at Google][starlark-github-repo], a dialect of Python for configuring the [Bazel build system][bazel-github]. [Kurtosis uses Starlark for the same purpose of configuring builds][starlark-explanation], except that we're building a distributed application rather than binaries or JARs.
+In this section, we created a `main.star` file that prints `Hello, world`. The `.star` extension corresponds to [the Starlark language developed at Google][starlark-github-repo], a dialect of Python for configuring the [Bazel build system][bazel-github]. [Kurtosis uses Starlark for the same purpose of configuring builds][starlark-explanation], except that we're building a distributed application rather than binaries or JARs.
 
-When you ran the Starlark, you got `Created enclave: quickstart`. An [enclave][enclaves-explanation] is a Kurtosis primitive that can be thought of as an *ephemeral test environment*, on top of Docker or Kubernetes, for a distributed application. The distributed applications that you define with Starlark will run inside enclaves. 
+When you ran `main.star`, you got `Created enclave: quickstart`. An [enclave][enclaves-explanation] is a Kurtosis primitive that can be thought of as an *ephemeral test environment*, on top of Docker or Kubernetes, for a distributed application. The distributed applications that you define with Starlark will run inside enclaves. If you'd like, you can tear down your enclave and any of their artifacts by running: `kurtosis clean -a` (more on the `kurtosis clean` command [here][kurtosis-clean-reference]).
 
 Enclaves are intended to be easy to create, easy to destroy, cheap to run, and isolated from each other. Use enclaves liberally!
 
+**In this section, you've experienced just how quickly and effortlessly Kurtosis can be used to generate and tear down ephemeral test environments on top of Docker or Kubernetes.**
```

Comment:
Did we see that though? Feels too early to say that.

Reply:
A few reasons why I think this is valid and should stay:
1. The user _did_ in fact see this: they wrote some Starlark and ran that code inside an isolated container and we showed the user how to (optionally) tear down their enclave using `kurtosis clean`. It's not a huge "wow" moment or something revolutionary, but we used this section to introduce the user to how quickly + easily it was to get an ephemeral environment up and running
2. For consistency's sake, every other section had a 1-2 sentence conclusion recapping a Kurtosis ability.

Reply:
MY PROBLEM: it feels like it's reaching too far to say anything about environments given that they've ONLY printed "Hello, world". They haven't truly spun up an environment, nor tore it down, nor actually done anything iwth containers at all.

Reply:
Got it. You've convinced me. 

```suggestion
```